approvals/src/main/java/no/sikt/nva/approvals/domain/ApprovalService.java

```diff
@@ -17,5 +18,5 @@ Approval getApprovalByNamedIdentifier(NamedIdentifier namedIdentifier)
         throws ApprovalNotFoundException, ApprovalServiceException;
 
     Approval updateApprovalIdentifiers(UUID approvalId,  Collection<NamedIdentifier> namedIdentifiers)
-        throws ApprovalNotFoundException, ApprovalConflictException, ApprovalServiceException;
+        throws ApprovalNotFoundException, ApprovalConflictException, ApprovalServiceException, RepositoryException;
 }
```

Comment:
Using exceptions as flow control is a new pattern we don't use in other services and warrants a discussion in the team I think. In any case, does the service really need to throw every single exception? Shouldn't it be abstracting these away into something generic so the handler doesn't need to know about repository specifics?